linera-core/src/worker.rs

```diff
@@ -88,6 +88,14 @@ mod metrics {
             &["validator_name"],
         )
     });
+
+    pub static CHAIN_INFO_QUERIES: LazyLock<IntCounterVec> = LazyLock::new(|| {
+        register_int_counter_vec(
```

Comment:
if we had `register_int_counter` (no `_vec`) then we wouldn't need the `.with_label_values(&[])`